src/front/app/routes.js

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
-import { Route } from 'react-router';
-import THSplash from './components/THSplash';
+import { Route, IndexRoute } from 'react-router';
+import Login from './components/login/Login';
+import Challenges from './components/challenges';
 
 export default () => (
-    <Route path="/" component={THSplash} />
+    <Route path="/">
+        <IndexRoute component={Login} />
+        <Route path="challenges" component={Challenges} />
```

Comment:
I suggest that we should has no default route, instead got something like ```/login``` and ```/main``` base routes